packages/unlock-js/src/web3Service.ts

```diff
@@ -963,4 +964,49 @@ export default class Web3Service extends UnlockService {
       return ''
     }
   }
+
+  /**
+   * Set signer for `Password hook contract`
+   */
+  async setPasswordHookSigner(
```

Comment:
This should be in WalletService as it modifies the state.

Reply:
moved to `WalletService `